06_puppy/selva/syncstore.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"sync"
+)
+
+// SyncStore struct
+type SyncStore struct {
+	sync.Mutex
```

Comment:
You've embedded a `Mutex` into `SyncStore`, but you're not using it. It is great that you have it, because for this code to work properly it is needed. Using it properly will require some learning from the later slides (`defer` from Errors, to unlock the mutex), and of course the mutex is not really needed until you introduce concurrency (after Errors).

So either:
1) Remove the embedded `sync.Mutex`, or
2) Add the locking that the mutex provides, or
3) Add a comment here that says that locking is needed for proper concurrent operation and that it is coming soon (a TODO comment). You should probably combine that with option 1.

The last option is a good one because it tells the reviewer that you are aware of what is needed for a correct implementation, but that it is for a future change. This makes it easier to review as I don't need to add comments about that because I then have the confidence that the code is going in the right direction.